django2/files/routes.yaml

```diff
@@ -0,0 +1,13 @@
+# The routes of the project.
```

Comment:
This file needs to be inside the `.platform` directory under `files`.

Reply:
Oh, I have copied it to the wrong path. fixed.